### PR TITLE
Kbuild: Convert DEBUG_PRINT_ON_UART1 to kconfig

### PR DESCRIPTION
**Pull request by jonasdn**

### CODE REVIEW

File: src/modules/src/system.c [hunk 1]
```diff
@@ -171,8 +171,8 @@ void systemTask(void *arg)
   queueMonitorInit();
 #endif
 
-#ifdef ENABLE_UART1
-  uart1Init(9600);
+#ifdef CONFIG_DEBUG_PRINT_ON_UART1
+  uart1Init(115200);
```
Comment (knmcguire) [inline src/modules/src/system.c:175]: Ah yeah good catch. Don't know why this was different then uart2 before.

File: Kconfig [hunk 2]
```diff
@@ -24,6 +24,14 @@ config DEBUG_QUEUE_MONITOR
     help
         Enable the queue monitoring functionality.
 
+config DEBUG_PRINT_ON_UART1
+    bool "Switch DEBUG_PRINT to UART1"
+    default n
+    help
+      Switch output of DEBUG_PRINT from radio to UART1 with 115200 baudrate.
+      Note! This will interfere with several expansion decks, see:
+      https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck-pin-allocation
```
Comment (knmcguire) [inline Kconfig:33]: it's a bit dangerous to put links in these places. These tend to get forgotten very easily, as we noticed lately 😄 wonder if we should make a short link to the expansiondecks instead. Or just refer them to the website and search deck pin allocation?

Comment (knmcguire) [reply Kconfig:33]: With forgotten, I mean if we every change the urls of these things and such

Comment (jonasdn) [reply Kconfig]: Agreed, pushed a commit to make it a bit safer.

**Review by knmcguire — APPROVED** [review]

**Comment by knmcguire** [comment]
perfect. thanks!

**Review by tobbeanton — COMMENTED** [review]
All looks very nice. I really like that we can use the dependency to prevent decks from compiling at the same time.

**Pull request merged**